static/scss/flex/article.scss

```diff
@@ -101,24 +92,24 @@ article section.page {
         }
     }
 
-
-
     code {
         font-family: 'RobotoMono', monospace
     }
 
     p>code,
     li code,
     table code {
-        padding: 0 .25rem
+        padding: 0 .25rem;
+        word-break: break-word;
```

Comment:
lets long words (like urls) break to wrap, instead of extending off the screen in an unreadable way